spec/std/process_spec.cr

```diff
@@ -239,4 +248,54 @@ describe Process do
       Process.find_executable("some_very_unlikely_file_to_exist").should be_nil
     end
   end
+
+  # This doesnt work in the linux test environment due to the test environment
+  # FIXME: Discover the reason why `LibC.getgrgid(2000)` does not return a user.
+  {% if !flag?(:linux) %}
```

Comment:
We can't really accept this without a fix for linux. I'll test this to see if it's just the CI environment or something else.

Reply:
I am going to try and run it again without the check and see if its changed since I originally wrote it.

Reply:
I've run it on my laptop on archlinux and they pass fine except for the one below.